particles/MeshParticles.hpp

```diff
@@ -17,12 +17,12 @@ namespace particles
 struct TexturedMeshForParticles final : score::gfx::Mesh
 {
     // Generate our mesh data
-    struct meshdata myData = getmesh();
-    const std::vector<float> mesh = myData.values;
+    struct meshdata myData;
```

Comment:
`struct meshdata` is C, in C++ we just write `meshdata`, never struct outside of a type definition / declaration.